GUI/Main.cs

```diff
@@ -1011,11 +1011,10 @@ private void exportModListToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void manageKspInstancesMenuItem_Click(object sender, EventArgs e)
         {
-            Instance.Manager.ClearAutoStart();
             var old_instance = Instance.CurrentInstance;
             var result = new ManageKspInstances(!actuallyVisible).ShowDialog();
             if (result == DialogResult.OK && !Equals(old_instance, Instance.CurrentInstance))
-                Instance.CurrentInstanceUpdated(false);
+                Instance.CurrentInstanceUpdated(true);
```

Comment:
I guess `onStartup` was a bad name for that parameter since this is clearly not startup. Would you mind renaming it, something like `allowRepoUpdate`?